tests/rafter/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/kyma-project/kyma/tests/rafter
+
+go 1.15
+
+require (
+	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/hashicorp/golang-lru v0.5.3 // indirect
+	github.com/imdario/mergo v0.3.8 // indirect
+	github.com/kyma-project/rafter v0.0.0-20191209102906-e1e7b47eddf0
+	github.com/onsi/gomega v1.10.1
+	github.com/pkg/errors v0.9.1
+	github.com/vrischmann/envconfig v1.3.0
+	golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6 // indirect
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+	google.golang.org/appengine v1.6.5 // indirect
+	k8s.io/api v0.17.11
+	k8s.io/apimachinery v0.17.12-rc.0
```

Comment:
bump all those k8s.io deps to v0.17.14

Reply:
:)

Reply:
`https://github.com/kubernetes/utils` is not possible to set fixed version, they don't release it.